portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/journal/structure/WebContentStructures.testcase

```diff
@@ -599,6 +599,16 @@ definition {
 			structureName = "Basic Web Content");
 	}
 
+	@description = "This is a test for LPS-125933. Create, read, update and delete a Structure with all possible fields."
+	@ignore = "true"
+	@priority = "5"
+	test ViewAndEditAsWellAsDeleteStructureWithTitleAndAllFields {
+		property portal.acceptance = "false";
```

Comment:
Maybe we can add these P5 issues to acceptance. What do you think?

Reply:
see below